WTWSMS/common/traits/WtWSMS_traits.txt

```diff
@@ -141,11 +141,14 @@ legendary_leader = {
 
 the_prophet = {
 	
-	muslim_opinion = 25
-	martial = 6
+	muslim_opinion = 100
+	martial = 12
```

Comment:
There are different kinds of opinions though. Even though you have your founder as your liege, you should get upset if he tries to touch crown authority or imprison you for no reason. 25 is largely sufficient.